packages/asc-core/src/components/GlobalStyle/GlobalStyle.ts

```diff
@@ -13,4 +13,8 @@ export default createGlobalStyle`
   [aria-hidden="true"] {
     display: none;
   }
+  
+  *:focus{
```

Comment:
why? are there elements that shouldn't have a focus style? we always override the outline on some elements so this seems a bbit pointless?

Reply:
I think setting out the default focus is e good thing while the way is displayed in different browsers is different. Each component that has a focus, wil implement the Amsterdam styled focus